app/forms/sections/guarantor/GuarantorVatFormProvider.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.sections.guarantor
+
+import forms.ONLY_ALPHANUMERIC_REGEX
+import forms.mappings.Mappings
+import play.api.data.Form
+
+import javax.inject.Inject
+
+class GuarantorVatFormProvider @Inject() extends Mappings {
+
+  def apply(): Form[String] =
+    Form(
+      "value" -> text("guarantorVat.error.required")
+        .verifying(maxLength(12, "guarantorVat.error.length"))
```

Comment:
This is probably me being naive. However, in the regex it says the max length is 14 chars?

Reply:
hmmm, https://github.com/hmrc/emcs-tfe-create-movement-frontend/blob/main/app/forms/sections/firstTransporter/FirstTransporterVatFormProvider.scala has 12 although https://github.com/hmrc/emcs-tfe-create-movement-frontend/blob/main/app/forms/sections/transportArranger/TransportArrangerVatFormProvider.scala has 14. I believe the schema for this is 14 (just checking that)

Reply:
Schema says `  <xs:simpleType name="VatNumberType">
    <xs:annotation>
      <xs:documentation>
        <doc:description value="VAT Number" />
      </xs:documentation>
    </xs:annotation>
    <xs:restriction base="xs:token">
      <xs:maxLength value="14" />
      <xs:pattern value=".{1,14}" />
    </xs:restriction>
  </xs:simpleType>`

Reply:
(or whatever country code)

Reply:
Updated in https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/84/commits/6d0fd9e7bfd6a21b0e37acc5510ecc31c5ed5b0b

Reply:
We can always change it back once the demo is done